languages.yaml

```diff
@@ -291,6 +291,56 @@ CoffeeScript:
     ansi:
       - red
     chip: "#244776"
+ColdFusion:
+  type: programming
+  ascii: |
+    {0}CfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCf
+    {0}Cf                                    Cf
+    {0}Cf                                    Cf
+    {0}Cf                         CfCfCfCf   Cf
+    {0}Cf       CfCfCfCfCf     CfCfCfCfCf    Cf
+    {0}Cf      CfCfCfCfCf     CfCf           Cf
+    {0}Cf     CfCf           CfCf            Cf
+    {0}Cf    CfCf         CfCfCfCfCfCf       Cf
+    {0}Cf    CfCf            CfCf            Cf
+    {0}Cf    CfCf            CfCf            Cf
+    {0}Cf     CfCf           CfCf            Cf
+    {0}Cf      CfCfCfCfCf    CfCf            Cf
+    {0}Cf       CfCfCfCfCf   CfCf            Cf
+    {0}Cf                                    Cf
+    {0}Cf                                    Cf
+    {0}CfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCf
+  colors:
+    ansi:
+      - Cyan
+    hex:
+      - "#DFF0FD"
+    chip: "#ed2cd6"
+ColdFusionScript:
+  type: programming
+  ascii: |
+    {0}CfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCf
+    {0}Cf                                    Cf
+    {0}Cf                                    Cf
+    {0}Cf                         CfCfCfCf   Cf
+    {0}Cf       CfCfCfCfCf     CfCfCfCfCf    Cf
+    {0}Cf      CfCfCfCfCf     CfCf           Cf
+    {0}Cf     CfCf           CfCf            Cf
+    {0}Cf    CfCf         CfCfCfCfCfCf       Cf
+    {0}Cf    CfCf            CfCf            Cf
+    {0}Cf    CfCf            CfCf            Cf
+    {0}Cf     CfCf           CfCf            Cf
+    {0}Cf      CfCfCfCfCf    CfCf            Cf
+    {0}Cf       CfCfCfCfCf   CfCf            Cf
+    {0}Cf                                    Cf
+    {0}Cf                                    Cf
+    {0}CfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCfCf
+  colors:
+    ansi:
+      - Cyan
+    hex:
+      - "#DFF0FD"
+    chip: "#ed2cd6"
```

Comment:
I agree with @spenserblack's suggestion that you could try creating a spin-off of the original logo (similar to what was done with Fortran) for your submission. However, if you're not feeling inspired at the moment, it's perfectly fine to submit only one entry for Coldfusion and leave the logo variation for another time.